### ISSUE TITLE
feat(lagrangian): footprint emulator with adjoint calibration and emulator-based inversion (Steps 3–4)

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/plumax#90]
## Problem / Request
Backward footprints cost `O(n_obs · N_particles)` per overpass; at basin scale with daily TROPOMI that is the bottleneck the whole modeling cycle exists to remove. Tier II is the first tier the roadmap says must be emulated (Step 3), with the emulator then substituted into the existing closed-form inversions (Step 4). Nothing in the repo does emulation yet; this is the first one and sets the pattern (`forward(params, met) → observations` interface, calibration against the reference model).

## User Story
> As a user running daily inversions, I want a trained footprint emulator `F_emu(met, receptor) ≈ F` and an inversion that uses it, validated against the particle model, so a basin-day inverts in seconds.

## Proposed API
```python
# src/plumax/lagrangian/emulator.py
class FootprintEmulator(eqx.Module):
    def __call__(self, met_context: jax.Array, receptor_xy: jax.Array) -> Float[Array, "N_cells"]: ...

def generate_training_set(key, *, met_sampler, receptors, grid, n_cases: int, n_particles: int) -> tuple[jax.Array, jax.Array]: ...
def train_footprint_emulator(model, data, *, steps, lr) -> FootprintEmulator: ...
def emulated_footprint(emu: FootprintEmulator, met_context, receptors) -> Float[Array, "N_obs N_cells"]: ...

# src/plumax/lagrangian/emulator_adjoint_test.py  (test helper, importable)
def adjoint_relative_error(emu, reference_F, *, met_context, receptors) -> float: ...
```

## Implementation Steps
- [ ] Add `src/plumax/lagrangian/emulator.py`; architecture v1 = per-receptor MLP/UNet on a normalised met context producing a sparse footprint patch (log-space output, softplus)
- [ ] Training set generator sampling met from the *operational* distribution, not the training bins (design validation note)
- [ ] `emulated_footprint` returns the same `(N_obs, N_cells)` layout `linear_gaussian_inversion` expects — Step 4 is then zero new inversion code
- [ ] Adjoint calibration harness: compare `jax.grad` through the emulator with `jax.grad` through `compute_footprint` on held-out met
- [ ] Record the hard test's threshold (< 5 % relative Frobenius / operator-norm error) in the module docstring

## Definition of Done
- [ ] ‖F_emu − F‖_F / ‖F‖_F < 5 % on held-out met from the operational distribution
- [ ] Step 4 posterior on a synthetic source matches the Step 2 posterior in mean and covariance within MC error
- [ ] `make test && make lint && make typecheck` green (training tests `@pytest.mark.slow`)

## Testing
- [ ] `tests/lagrangian/test_emulator.py::test_residual_below_5pct` (`slow`)
- [ ] `tests/lagrangian/test_emulator.py::test_step4_matches_step2_posterior` (`slow`)
- [ ] `tests/lagrangian/test_emulator.py::test_adjoint_calibration`

## Relationships
- Parent (theme epic): #71
- Blocked by: C-grid-aware wind interpolator (training data needs gridded wind)
- Related: Tier III FV emulator (same harness pattern), Tier IV stacked emulator
